exec-unittests

```diff
@@ -161,43 +166,57 @@ if [[ -z "${subtst}" || "${subtst}" == "python" ]]; then
     fi
 fi
 
-_subtst_list="tool-scripts/datalog tool-scripts/postprocess tool-scripts util-scripts bench-scripts"
+_agent_subtst_list="tool-scripts/datalog tool-scripts/postprocess tool-scripts util-scripts bench-scripts"
 
 _para_jobs_file="${_toxenvdir}/agent-legacy-jobs.lis"
 trap "rm -f ${_para_jobs_file}" EXIT INT TERM
 
 for _major in ${major_list}; do
-    # Verify the Agent or Server Makefile functions correctly.
-    if [[ "${_major}" == "agent" || "${_major}" == "server" ]]; then
-        verify_make_source_tree ${_major} || rc=1
-    fi
-
     if [[ "${_major}" == "agent" ]]; then
```

Comment:
With this change, there is now no common code in the body of the loop -- it's segregated by `agent`/`client`/`server` -- in which case there's not much point in having the loop.  If you're amenable, instead we can just say "if `agent` is in `$_major`, then do the agent stuff", then "if `server` is in `$_major`, then do the server stuff", and so on.  (If you're interested and don't know how to check for a value in a list, I'll see what I can come up with....)